nodebuilder/daser/config.go

```diff
@@ -0,0 +1,46 @@
+package daser
+
+import (
+	"errors"
+	"time"
+)
+
+var (
+	ErrNegativeInterval = errors.New("nodebuilder/daser: interval must be positive")
+)
+
+type Config struct {
+	//  samplingRange is the maximum amount of headers processed in one job.
+	SamplingRange uint64 // TODO(@derrandz): Question to @vlad, why is this uint64?
```

Comment:
it was convenient to have it as `uint64` constant, because it was compared with other `uitn54` values in code later on. For config it is better to have all values of same type (`uint`)) and convert it internally in daser where appropriate.

Reply:
I agree about the point of having all types be `uint`. We should also scrutinize in the other variables that you mentioned were `uint64` and revise if they should remain so.

Tagging @Wondertan